recipe/build_folly.sh

```diff
@@ -50,12 +52,27 @@ if [[ "${target_platform}" == "linux-ppc64le" ]]; then
     CMAKE_ARGS="${CMAKE_ARGS} -DHAVE_VSNPRINTF_ERRORS_EXITCODE=0 -DHAVE_VSNPRINTF_ERRORS_EXITCODE__TRYRUN_OUTPUT=''"
 fi
 
-cmake ${CMAKE_ARGS} -Wno-dev -GNinja -DBUILD_SHARED_LIBS=ON ..
+# Build a shared library for the "folly" package or
+# build a static library for the "folly-static" package.
+if [[ ! -z "${PKG_NAME+x}" && "${PKG_NAME}" == "folly" ]]
+then
```

Comment:
`PKG_NAME` should always be defined.
```suggestion
if [[ "${PKG_NAME}" == "folly" ]]; then
```